pxl/compress.py

```diff
@@ -0,0 +1,35 @@
+from pathlib import Path
+from PIL import Image
+from resizeimage import resizeimage
+
+# TODO: These values are picked at random wihtout much thought
+WIDTHS = [1600, 800, 400, 250]
+
+
+def compress_image(local_filename: Path) -> [Path]:
+    """Compresses the image to different sizes and saves them to /tmp, returns a list of paths"""
+
+    image_paths = []
+    with open(str(local_filename)) as file:
+        with Image.open(local_filename) as image:
+            # Copy the original image to /tmp
+            path = Path("/tmp").joinpath(local_filename.name)
+            image.save(path)
```

Comment:
Do we need to copy the original image?

Reply:
I thought you had the intention to keep the original image for download purposes. It seemed logical to have all the paths that the function returns, point to the same folder.